Lab.tests/lab02.cs

```diff
@@ -0,0 +1,67 @@
+﻿using Lab;
+using System;
+using Xunit;
+
+namespace Lab.Tests
+{
+    public class lab02
+    {
+        [Fact]
+        public void ViewBalanceTest()
+        {
+            // Arrange
+            double balance = Program.balance;
+            // Act
+            double result = Program.ViewBalance(balance);
```

Comment:
1. I would expect this to be just `ViewBalance()`. I don't get to specify what my balance is!
2. We should use `decimal` instead of `double`!